### PR TITLE
KAFKA-5507: Check classpath empty in kafka-run-class.sh

### PR DESCRIPTION
**Pull request by evis**

### CODE REVIEW

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/5646/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/5660/
Test PASSed (JDK 7 and Scala 2.11).

**Comment by evis** [comment]
@ijuma @guozhangwang could you review this PR, please?

File: bin/kafka-run-class.sh
Comment (ijuma) [inline bin/kafka-run-class.sh]: It seems like it would be better to recommend something like `./gradlew jar -PscalaVersion=$SCALA_VERSION` or something.

Comment (evis) [reply bin/kafka-run-class.sh]: Fixed as you suggested.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk8-scala2.12/6061/
Test PASSed (JDK 8 and Scala 2.12).

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/kafka-pr-jdk7-scala2.11/6076/
Test PASSed (JDK 7 and Scala 2.11).

**Review by ijuma — APPROVED** [review]
LGTM, thanks.

**Review by ijuma — COMMENTED** [review]
Because we have no tests for this, this needs manual testing before it can be merged.

**Comment by evis** [comment]
Who should do manual testing? Me or someone of commiters?

**Comment by ijuma** [comment]
One of the committers, hopefully I'll get to it soon.

File: bin/kafka-run-class.sh
Comment (ijuma) [inline bin/kafka-run-class.sh:144]: I just tested this and it doesn't actually print the $SCALA_VERSION variable as I expected. I thought that the following would ensure it's set:

```sh
if [ -z "$SCALA_VERSION" ]; then
  SCALA_VERSION=2.11.11
fi
```

Can you please check?

Comment (evis) [reply bin/kafka-run-class.sh:144]: Yes, it doesn't print $SCALA_VERSION variable. I thought, that if this variable is unset, then message would be strange, with such command: `./gradlew jar -Pscala_version`. Do you suggest to set variable, if it's unset? I should use double quotes in `echo` then.

Comment (ijuma) [reply bin/kafka-run-class.sh:144]: The script itself sets the variable if unset by the user. So, it seems safe to print it.

Comment (evis) [reply bin/kafka-run-class.sh:144]: Thanks, I missed it. I will change it, so that `echo` will print `$SCALA_VERSION` variable value.

**Review by ijuma — APPROVED** [review]
LGTM, I fixed the minor issue before merging to trunk.

**Pull request closed without merging**
